### PR TITLE
Fix `Dashboard` breadcrumb view URL for user profile

### LINKED ISSUE #709: Page Not Found while opening Dashboard
### How to reproduce
1. Sign up at junction through GitHub.
2. Click on Username once logged In.
3. Click on Profile.
4. Now click on _Dashboard_ (**In navigation panel, on the left side**).
5. Demo below
bug

### Expected Behaviour
Currently the Dashboard link points to : [https://in.pycon.org/cfp/profiles/edit/user-proposals-list]().
Instead it should (or may) point to [https://in.pycon.org/cfp/profiles/]()

### PR DESCRIPTION
**Pull request by abhishekmishragithub**
Update view URL for `Dashboard` breadcrumb name under user profile

This Fixes #709

### CODE REVIEW

File: junction/templates/profiles/userprofile.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 {% block breadcrumbs %}
     {{ block.super }}
-    {% breadcrumb "Dashboard" "user-proposals-list" %}
+    {% breadcrumb "Dashboard" "/profiles" %}
     {% breadcrumb "Profile" "user-profile"%}
```
Comment (sayanchowdhury) [inline junction/templates/profiles/userprofile.html:21]: Given the second param is dummy, and of no use can you add a comment regarding that?

File: junction/templates/profiles/dashboard.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 {% block breadcrumbs %}
     {{ block.super }}
-    {% breadcrumb "Dashboard" "user-proposals-list" %}
+    {% breadcrumb "Dashboard" "/profiles" %}
```
Comment (sayanchowdhury) [inline junction/templates/profiles/dashboard.html:20]: Same for here, add the comment for second argument.

File: junction/templates/profiles/userprofile.html (hunk 1 above)
Comment (abhishekmishragithub) [reply junction/templates/profiles/userprofile.html:21]: sure, will work on that.

File: junction/templates/profiles/dashboard.html (hunk 2 above)
Comment (abhishekmishragithub) [reply junction/templates/profiles/dashboard.html]: Done.

File: junction/templates/profiles/userprofile.html (hunk 1 above)
Comment (abhishekmishragithub) [reply junction/templates/profiles/userprofile.html]: Done.

**Comment by abhishekmishragithub** [comment]
> LGTM. Better if you squash the commits.

Done.
Please merge.

**Pull request merged**